packages/libs/user-datasets/src/lib/Components/UserDatasetStatus.tsx

```diff
@@ -56,7 +53,7 @@ export default function UserDatasetStatus(props: Props) {
     : 'exclamation-circle';
   const children = <Icon className="StatusIcon" fa={faIcon} />;
   const visibleContent = props.useTooltip ? (
-    <Tooltip content={content}>{children}</Tooltip>
+    <Tooltip title={content}>{children}</Tooltip>
```

Comment:
This tooltip is still not firing off in the `Status` column of the `My Data Sets` table. I think this component just needs refactoring. I think if we render the Tooltip above the Link component, it'll work just fine. If you don't mind, could you open an issue for this and I'll do the follow up work in the VDI client branch after merging this work into it? It'll create a big enough merge conflict in that component such that it makes just as much sense for me to handle it!

Reply:
Here's the new ticket! https://github.com/VEuPathDB/EdaNewIssues/issues/776